docs/source/api/consistenthash.rst

```diff
@@ -34,17 +34,42 @@ Request Structure
 :requestpath: The request path to use to test the regular expression against
 :cdnid:       The unique identifier of a CDN that will be used to query for an active Traffic Router
 
+.. code-block:: http
+	:caption: Request Example
+
+	POST /api/1.5/consistenthash HTTP/2
+	Host: trafficops.infra.ciab.test
+	User-Agent: curl/7.54.0
+	Accept: */*
+	Cookie: mojolicious=...
+	Content-Length: 80
+	Content-Type: application/x-www-form-urlencoded
+
+	{"regex":"/.*?(/.*?/).*?(m3u8)","requestpath":"/test/path/asset.m3u8","cdnid":2}
+
 Response Structure
 ------------------
 :resultingPathToConsistentHash: The resulting path that Traffic Router will use for consistent hashing
 :consistentHashRegex:           The regex used by Traffic Router derived from POST 'regex' parameter
 :requestPath:                   The request path used by Traffic Router to test regex against
 
-.. code-block:: json
+.. code-block:: http
 	:caption: Response Example
 
+	HTTP/2 200
```

Comment:
Unfortunately this is causing compilation errors because the Pygments library used for highlighting syntax (like HTTP) does not have support for HTTP/2 (I have [an open issue](https://bitbucket.org/birkenfeld/pygments-main/issues/1486/lexing-support-for-http-2) about it). If you just change `HTTP/2` to `HTTP/1.1` (in both the request and the reply), and add a 'reason' text (i.e. `200 OK` not just `200` - don't ask, I don't know why) the lexer will be happy.

Reply:
okay, made the change